app/javascript/app/data/constants.js

```diff
@@ -39,18 +39,6 @@ export const GHG_CALCULATION_OPTIONS = {
   }
 };
 
-export const CHART_TYPES = {
```

Comment:
These variables shouldn't be deleted. They are used in the code

Reply:
Thanks a lot for checking @Bluesmile82 ! Sorry just saw your comments here and yes I guess the only change needed is the "resources" in the URL

Reply:
No worries! I'll close this PR and make the change on a different one. Thanks!